### PR TITLE
Update Italian (it_IT) translation

### PR DESCRIPTION
**Pull request by farsil**
# Description

Updates the Italian translation to include the latest changes to `CONFIG_SHELL_HISTORY_FILE`. Whitespace was trimmed from some of the lines.

# Manual testing

N/A

# Checklist

_Please tick the items as you have addressed them. Don't remove items; leave the ones that are not applicable unchecked._

I have:

- [x] followed the project's [contributing guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/dosbox-staging/dosbox-staging/blob/master/CODE_OF_CONDUCT.md).
- [ ] performed a self-review of my code.
- [ ] commented on the particularly hard-to-understand areas of my code.
- [ ] split my work into well-defined, bisectable commits, and I [named my commits well](https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages).
- [ ] applied the appropriate labels (bug, enhancement, refactoring, documentation, etc.)
- [ ] [checked](https://github.com/dosbox-staging/dosbox-staging/blob/main/scripts/compile_commits.sh) that all my commits can be built.
- [ ] confirmed that my code does not cause performance regressions (e.g., by running the Quake benchmark).
- [ ] added unit tests where applicable to prove the correctness of my code and to avoid future regressions.
- [ ] made corresponding changes to the documentation or the website according to the [documentation guidelines](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md).
- [ ] [locally verified](https://github.com/dosbox-staging/dosbox-staging/blob/main/DOCUMENTATION.md#previewing-documentation-changes-locally) my website or documentation changes.

### CODE REVIEW

File: contrib/resources/translations/it.lng [hunk 1]
```diff
@@ -1348,16 +1348,16 @@ Nessuna ROM è attualmente caricata
 s
 .
 :MT32_ROMS_LABEL
-ROM  MT-32  
+ROM  MT-32
```
Comment (farsil) [inline contrib/resources/translations/it.lng:1351]: Not sure if the trailing whitespace in these labels is intentional, if it is I'll roll back.

Comment (kcgen) [reply contrib/resources/translations/it.lng:1351]: Yes; the whitespace isn't needed and can be stripped; thanks for this cleanup.

(When padding or aligning, we'll do it programmatically using `%` printf format specifiers.)

**Review by kcgen — APPROVED** [review]

**Comment by kcgen** [comment]
(Tagged w/ backport for merge to 0.81.x, and to `main`)

**Comment by johnnovak** [comment]
Thanks @farsil 🎉

**Pull request merged**